test/types/bytes/bytesFormattedIO.chpl

```diff
@@ -0,0 +1,22 @@
+var b: _bytes = "some\t\x00bytes with € (the euro char)";
+writeln("Length :", b.numBytes);
+writeln();
+writef("<%|36s>\n", b);
+writeln();
+writef("<%|*s>\n", 36, b);
+writeln();
+writef("<%|0S>\n", b);
+writeln();
+writef("<%|1S>\n", b);
+writeln();
+writef("<%|2S>\n", b);
+writeln();
+writef("<%|4S>\n", b);
+writeln();
+writef("<%|8S>\n", b);
+writeln();
+writef("<%|*S>\n", 1, b);
+writeln();
+writef("<%|vS>\n", b);
+writeln();
+writef("<%ht>\n", b);
```

Comment:
Can you add some %s arguments with things like field widths? Maybe they fail today... but it would be good to have testing of them.

Reply:
Also it'd be nice to have the text tests in a separate test, so that the output .good can more easily be examined for that case.